Selenium/SeleniumTestProject.cs

```diff
@@ -0,0 +1,130 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Selenium;
+
+[TestFixture]
+public class Tests
+{
+    private IWebDriver _driver;
+
+    [SetUp]
+    public void Setup()
+    {
+        _driver = new ChromeDriver();
+    }
+
+    [Category("Smoke Test")]
+    [Category("Positive")]
+    [Property("Severity", "High")]
+    [Test]
+    public void CheckKermifkoSite()
+    {
+        _driver.Navigate().GoToUrl("https://kermi-fko.ru/raschety/Calc-Rehau-Solelec.aspx");
+
+        var floorWidthField = _driver.FindElement(By.Id("el_f_width"));
+        floorWidthField.SendKeys("13");
+
+        var floorLengthField = _driver.FindElement(By.Id("el_f_lenght"));
+        floorLengthField.SendKeys("10");
+
+        var roomType = _driver.FindElement(By.Id("room_type"));
+        var listOfRoomTypes = new SelectElement(roomType);
+        listOfRoomTypes.SelectByIndex(2);
+
+        var heatingType = _driver.FindElement(By.Id("heating_type"));
+        var listOfHeatingTypes = new SelectElement(heatingType);
+        listOfHeatingTypes.SelectByValue("3");
+        Assert.AreEqual("Подогрев для комфорта", listOfHeatingTypes.SelectedOption.Text, "Smth went wrong...");
+
+        var heatLossField = _driver.FindElement(By.Id("el_f_losses"));
+        heatLossField.SendKeys("10");
+
+        var countButton = _driver.FindElement(By.CssSelector(".buttHFcalc"));
+        countButton.Click();
+
+        var floorCablePower = _driver.FindElement(By.Id("floor_cable_power"));
+        Assert.AreEqual("6", floorCablePower.GetAttribute("value"), "Smth went wrong...");
+
+        var specificFloorCablePower = _driver.FindElement(By.Id("spec_floor_cable_power"));
+        Assert.AreEqual("0", specificFloorCablePower.GetAttribute("value"), "Smth went wrong...");
+    }
+
+    [Category("Smoke Test")]
+    [Category("Positive")]
+    [Property("Severity", "High")]
+    [Test]
+    public void CheckMasterskayaPolaSite()
+    {
+        _driver.Navigate().GoToUrl("https://masterskayapola.ru/kalkulyator/laminata.html");
+
+        var inputFormsBy = By.CssSelector(".form-control.checknumber");
+        var inputForms = _driver.FindElements(inputFormsBy);
```

Comment:
Everywhere! Please, write by into find element 
As you did before : var specificFloorCablePower = _driver.FindElement(By.Id("spec_floor_cable_power"));